lib/bokken_web/templates/email/ninja_event_participation.text.eex

```diff
@@ -0,0 +1,8 @@
+Olá 
+
+Vimos por este meio confirmar se o ninja não vai participar na próxima sessão.
```

Comment:
```suggestion
Vimos por este meio confirmar que o ninja não vai participar na próxima sessão.
```
You should add the ninja's name, as there are parents with more than one ninja and it can happen that one makes it and the other doesn't.